pkg/pb/synthetic_monitoring/checks.proto

```diff
@@ -192,10 +192,18 @@ message Check {
 // CheckSettings provides the settings for exactly one type of check.
 message CheckSettings {
 	option (gogoproto.onlyone) = true;
-	PingSettings ping = 1 [(gogoproto.jsontag) = "ping,omitempty"];
-	HttpSettings http = 2 [(gogoproto.jsontag) = "http,omitempty"];
-	DnsSettings  dns  = 3 [(gogoproto.jsontag) = "dns,omitempty"];
-	TcpSettings  tcp  = 4 [(gogoproto.jsontag) = "tcp,omitempty"];
+	PingSettings ping 						= 1 [(gogoproto.jsontag) = "ping,omitempty"];
+	HttpSettings http 						= 2 [(gogoproto.jsontag) = "http,omitempty"];
+	DnsSettings  dns  						= 3 [(gogoproto.jsontag) = "dns,omitempty"];
+	TcpSettings  tcp  						= 4 [(gogoproto.jsontag) = "tcp,omitempty"];
+	TracerouteSettings traceroute = 5 [(gogoproto.jsontag) = "traceroute,omitempty"];
+}
+
+message TracerouteSettings {
+	int64 maxHops 	 		 = 1 [(gogoproto.jsontag) = "maxHops"]; 
+	int64 maxUnknownHops = 2 [(gogoproto.jsontag) = "maxUnknownHops"];
+	bool  ptrLookup 		 = 3 [(gogoproto.jsontag) = "ptrLookup"];
+	int64 hopTimeout 		 = 4 [(gogoproto.jsontag) = "hopTimeout"];
```

Comment:
Could we add some code comments inline, at the end of each line here? Like we do for `PingSettings` below 🙏

Reply:
These config options could be removed from this PR and go into a later PR potentially. In order to make this PR smaller, and easier to work on.

Reply:
I added the comments but left the config for now. Totally open to pulling them out, though